plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/button/index.tsx

```diff
@@ -48,12 +46,11 @@ const featurePluginSupport = {
 		__experimentalSelector:
 			'.wp-block-button.wc-block-components-product-button .wc-block-components-product-button__button',
 	} ),
-	...( typeof __experimentalGetSpacingClassesAndStyles === 'function' &&
-		! isFeaturePluginBuild() && {
-			spacing: {
-				margin: true,
-			},
-		} ),
+	...( typeof __experimentalGetSpacingClassesAndStyles === 'function' && {
```

Comment:
I kinda think if we're not calling this `fn` here we can probably remove the check right?